cmd/minishift/cmd/delete.go

```diff
@@ -36,6 +37,9 @@ var deleteCmd = &cobra.Command{
 }
 
 func runDelete(cmd *cobra.Command, args []string) {
+	instancedata.Instance().ReadInstanceData()
```

Comment:
@hferentschik 

There is one problem at the _root.go_ to create json file since we done have that path `.minishift/machines/minishift/foo.json` since VM is not up at this moment. 
Hence, we can't create json at root.go but we can read and populate instance in root.go after initialize if json file exists. This is use case for start -> stop -> start,    start -> stop -> delete.